KeepLayout.podspec

```diff
@@ -9,6 +9,35 @@ Pod::Spec.new do |s|
 
   s.source       = { :git => "https://github.com/Tricertops/KeepLayout.git", :tag => "v1.7.0" }
 
-  s.source_files = 'Sources'
   s.requires_arc = true
+
+  s.module_name = "KeepLayout"
+  s.default_subspec = 'Swift'
+
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
+  s.tvos.deployment_target = '9.0'
+
+  s.subspec 'ObjC' do |op|
+    op.source_files = 'Sources'
+    op.exclude_files = 'Sources/*.{swift}'
+    op.ios.deployment_target = '5.0'
+    op.osx.deployment_target = '10.7'
+    op.tvos.deployment_target = '9.0'
+    op.ios.framework = 'UIKit'
+    op.tvos.framework = 'UIKit'
+    op.osx.framework = 'Cocoa'
+    op.osx.exclude_files = 'Sources/UIViewController+KeepLayout.{h,m}', 'Sources/UIScrollView+KeepLayout.{h,m}'
+  end
+  s.subspec 'Swift' do |sp|
+    sp.dependency 'KeepLayout/ObjC'
+    sp.source_files = 'Sources/*.{swift}'
```

Comment:
Yes, they added with 33 line. 
I have next struct in the Pods project: 
Pods
    KeepLayout
          ObjC
          SupportFiles
          Swift

Also I've create a test project with one swift and one ObjC target (I've posted pod file from it), and all works file